build-scripts/build_all_guides.py

```diff
@@ -15,8 +15,11 @@
 import threading
 import sys
 
-import ssg
-ElementTree = ssg.xml.ElementTree
+import ssg.build_guides
+import ssg.xccdf
+import ssg.utils
+import ssg.xml
+from ssg.xml import ElementTree
```

Comment:
how about using ssg.xml.ElementTree in the code? it's slightly more chatty but it helps new devs figure out where to get it. maybe it will prevent people from importing the python ElementTree

Reply:
@mpreisler Okay, to my knowledge all external uses of `ElementTree` in `build-scripts` have been replaced by `ssg.xml.ElementTree`.